File: DivinoLanches/ClientApp/src/models/SubTipoProdutoModel.ts
export class SubTipoProdutoModel {

    id: number
    tipoProduto: string
    nome: string
    ativo: boolean

    constructor(obj: any) {
        this.id = obj.id
        this.tipoProduto = obj.tipoProduto
        this.nome = obj.nome
        this.ativo = obj.ativo
        return this
    }
}

File: DivinoLanches/Models/SubTipoProdutoModel.cs
using System;

namespace DivinoLanches.Models
{
    public class SubTipoProdutoModel
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string TipoProduto { get; set; }
        public bool? Ativo { get; set; }
    }
}
